test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/ConfigurationManagerTests.cs

```diff
@@ -330,6 +331,43 @@ public void ResetLastKnownGoodLifetime()
             TestUtilities.AssertFailIfErrors(context);
         }
 
+        [Fact]
+        public void TestConfigurationComparer()
```

Comment:
>TestConfigurationComparer [](http://example.com/codeflow?start=20&length=25)

Test case where token endpoint differs but is considered duplicate because it's not part of the hash check?

Reply:
Also consider a test case which covers duplicate keys with same kid

Reply:
Added test